### PR TITLE
submitting project_one

### PR DESCRIPTION
**Pull request by fuzailkhan95**
fuzail.khan@lc.cuny.edu

### CODE REVIEW

File: project_one/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <img src="logo.png">
+            </div>
+            <ul>
+                <li id="grading-element-one" onclick="myFunctionOne()"><a id="grading-element-five" href="#">Call Us</a></li>
+                <li class="active"><a id="grading-element-five" href="index.html">Home</a></li>
+                <li><a id="grading-element-five" href="">Book A Room</a></li>
+                <li><a id="grading-element-five" href="about.html" id="grading-element-two">About</a></li>
+                
+                <li id="grading-element-three" onclick="myFunction()"><a href="#">Location</a></li>
+                <div id="grading-element-one-consequence"></div>
```
Comment (lehmanclass) [inline project_one/index.html]: this should start absent, not in the HTMl or DOM at all. Use `const new = document.createElement` to build it inside `myFunctionOne()`

File: project_one/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <header>
+        <div class="main">
+            <div class="logo">
+                <img src="logo.png">
+            </div>
+            <ul>
+                <li id="grading-element-one" onclick="myFunctionOne()"><a id="grading-element-five" href="#">Call Us</a></li>
+                <li class="active"><a id="grading-element-five" href="index.html">Home</a></li>
+                <li><a id="grading-element-five" href="">Book A Room</a></li>
+                <li><a id="grading-element-five" href="about.html" id="grading-element-two">About</a></li>
```
Comment (lehmanclass) [inline project_one/index.html]: you declare two `id=` in this `a` tag...kill one and this test will work as expected.

File: project_one/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                
+                <li id="grading-element-three" onclick="myFunction()"><a href="#">Location</a></li>
+                <div id="grading-element-one-consequence"></div>
+                    <!--<p id="grading-element-one-consequence"></p>-->
+                <div id="grading-element-three-consequence" class="hide">Miami, FL</div>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            </ul>
+            <form action="http://localhost:3000/form" name="grading-element-four" method="POST">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+<body>
+    <header>
+        <div class="main">
+            <div class="logo">
+                <img src="logo.png">
+            </div>
+            <ul>
+                <li id="grading-element-one" onclick="myFunctionOne()"><a id="grading-element-five" href="#">Call Us</a></li>
+                <li class="active"><a id="grading-element-five" href="index.html">Home</a></li>
+                <li><a id="grading-element-five" href="">Book A Room</a></li>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected...except you have multiple `id="grading-element-five"` declarations, which you should fix (only one allowed!)

**Comment by lehmanclass** [comment]
needs a few tweaks. Please share any extra credit work you do in a `README.md` file.

**Comment by fuzailkhan95** [comment]
Professor, I just pushed my final version of my project on Github. Would you be able to see it? Please let me know. Thank you.

**Pull request closed without merging**